test/CMakeLists.txt

```diff
@@ -20,6 +20,9 @@ if (BUILD_TESTS)
         target_include_directories(testrunner SYSTEM PRIVATE ${tinyxml2_INCLUDE_DIRS})
     endif()
     target_externals_include_directories(testrunner PRIVATE ${PROJECT_SOURCE_DIR}/externals/simplecpp/)
+    if (Boost_FOUND)
+        target_include_directories(testrunner SYSTEM PRIVATE ${Boost_INCLUDE_DIRS})
+    endif()
```

Comment:
Having to updated all the consumers when dependencies change is very fragile and error-prone. #7658 fixes that issue.

Reply:
Yes, but that should not be merged so close to the release. And fixing such a minor issue within implicitly in a major rework is not great.

Reply:
Its always a small fix while technical debt piles up. Its best to take the opportunity to fix the issue once and for all.